modules/data/log/tmpl/index.tmpl

```diff
@@ -0,0 +1,40 @@
+<? I18N znc-log ?>
+<? INC Header.tmpl ?>
+
+<? IF Files ?>
+<h3>File Browser</h3>
+<div class="sectionbg">
+<div class="sectionbody">
+<ul style="list-style: none;">
+<? LOOP Files ?>
+<li><a href="<? VAR ModPath TOP ?>?path=<? VAR Long ESC=HTML,URL ?>"><? VAR Short ?></a></li>
+<? ENDLOOP ?>
+</ul>
+</div>
+</div>
+<? ENDIF ?>
+
+<? IF Log ?>
+<h3>Log Viewer (Page <? VAR Page ?>)</h3>
+<div class="sectionbg">
+<div class="sectionbody">
+<code style="display: block; margin-bottom: 0.5em;">
+<? LOOP Log ?>
+<? VAR Line ?><br>
+<? ENDLOOP ?>
+</code>
+<? IF PrevPage || PrevPage == 0 ?>
+<button><a style="text-decoration: none;" href="<? VAR ModPath TOP ?>?path=<? VAR Path ESC=HTML,URL ?>&page=<? VAR PrevPage ESC=HTML,URL ?>">Prev</a></button>
+<? ENDIF ?>
+<? IF NextPage || NextPage == 0 ?>
+<button><a style="text-decoration: none;" href="<? VAR ModPath TOP ?>?path=<? VAR Path ESC=HTML,URL ?>&page=<? VAR NextPage ESC=HTML,URL ?>">Next</a></button>
+<? ENDIF ?>
+</div>
+</div>
+<? ENDIF ?>
+
+<? IF !Files && !Log ?>
+<p><b>Logs cannot be accessed from the Global Module.</b></p>
```

Comment:
That's not what I see in the condition one line above

Reply:
Ok. Fair enough. While it does what I needed it to do I guess I should add a variable from the module to signal this condition. Or is there another way to check the module type from the template?

Reply:
I don't think so. A variable would work. Did you test this with 2 other types?

Reply:
No I did not but I didn't see a reason to as the other 2 module types work the same anyway for what I was doing. I will work on this now.

Reply:
I have revised it to be more obvious about what the conditional is actually doing. I also restructured the other modules to an ELSE branch so it flows more logically.